l10n/en/products/vpn/more/vpn-or-proxy.ftl

```diff
@@ -0,0 +1,58 @@
+
+# Variables
+#   $url (url) - https://www.consumer.ftc.gov/articles/0014-tips-using-public-wi-fi-networks
+vpn-or-proxy-a-proxy-is-useful = A proxy is useful when you’re browsing the web on a public WiFi. When a proxy is enabled, it will stop eavesdroppers on the same network from spying on your browsing activity or reading your transactions on unencrypted sites. It sounds harmless, but public WiFi networks can be like a backdoor for <a href="{ $url }">hackers</a>.
+
+# Variables
+#   $fpn (url) - https://fpn.firefox.com
+vpn-or-proxy-firefox-private-network = <a href="{ $fpn }">{ -brand-name-firefox-private-network }</a> is an easy to install browser extension that provides a secure, encrypted tunnel to the web to protect your browser connection anywhere you use { -brand-name-firefox }. It’s fast and easy to turn on whenever you need it for extra security in your browser.
+vpn-or-proxy-vpns-devicelevel-protection = VPNs: device-level protection
+vpn-or-proxy-vpns-do-more = VPNs do more than proxies in that a proxy only protects what you do in your browser, whereas a VPN protects all your  traffic, including your browser, wherever you have a VPN installed and enabled. VPNs provide added security and privacy for all your online activity—an important consideration if you want to keep your activity to yourself and make it more difficult for data hungry trackers and ISPs to create a profile of you across all your devices, like your phone, computers and tablet.
+vpn-or-proxy-a-vpn-works = A VPN works by creating a secure “tunnel” between your device and the internet at large. It protects your privacy in two key ways:
+vpn-or-proxy-concealing-your-ip = Concealing your IP address, protecting your identity and obscuring your location.
+vpn-or-proxy-encrypting-your-traffic = Encrypting your traffic between you and your VPN provider so that no one on your local network can decipher or modify it.
+
+# Variables
+#   $mozvpn (url) - https://www.mozilla.org/products/vpn/
+vpn-or-proxy-a-vpn-also = A VPN also offers security on open and public WiFi connections. Open WiFi can be risky, and it’s impossible to be sure that someone else isn’t connecting to the same network to snoop on what you’re doing. Even if your traffic is encrypted, they can still see which sites you are visiting. And if you’re using an app that doesn’t have encryption — and even today, many don’t — then they can see everything you are doing in that app. { -brand-name-mozilla-vpn } is a fast, secure, trustworthy service that can help close the security gap for you. <a href="{ $mozvpn }">{ -brand-name-mozilla-vpn }</a> also lets you choose your “location” (where your traffic appears to be coming from) from more than 30 countries.
```

Comment:
Should 30 be a variable here? https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L1569

Reply:
Good call. I did the same on the What Is a VPN page as well.